client.go

```diff
@@ -1475,4 +1484,9 @@ type baseCommit struct {
 type baseCommitPartial struct {
     Dan string `xml:"device-and-network,omitempty"`
     Pao string `xml:"policy-and-objects,omitempty"`
+    Admin *baseCommitPartialAdmin `xml:"admin,omitempty"`
+}
+
+type baseCommitPartialAdmin struct {
```

Comment:
Can remove this when switching to `util.MemberType`.

Reply:
Sure, I updated.